### DISCUSSION TITLE
A new slider for a dark overlay to further dim the screen than the monitor's brightness

### DISCUSSION THREAD

**Discussion by IbyA** [xanderfrangos/twinkle-tray#780] [Ideas]
Hi,

I’ve been using Twinkle Tray, and it’s fantastic! However, after switching to a new monitor, I noticed that even at its lowest brightness settings, the screen remains too bright—especially in dark environments. I’m wondering if it’s possible to add an additional slider option that overlays the entire screen with a dark image. This way, we could effectively darken the display further.

To illustrate my idea, imagine a feature similar to what you find in Photoshop: You take a screenshot/image in one layer, then add a dark shape layer on top of it. Finally, you apply a “multiply” filter to that layer, adjusting the opacity to control the darkness.

What do you all think? Would this be a useful addition to Twinkle Tray?

**Comment by Stk252** [comment]
twinkle-tray is already fantastic, but this feature would be really great.

I also have monitors that at the lowest brightness setting are still too bright (HP Omen 27u 4K).
I'm currently using a combination of Twinkle Tray and Dimmer. 

Dimmer is a good tool, I use it at night when twinkle-tray is not enough because of my monitors brightness, but it would be great to have the entire functionality (brightness control + overlay) on a single tool.

**Comment by philippwallrafen** [comment]
Yesss! I'm using Dimmer from Nelson Pires and Twinkle Tray simultaniously every day. Dimmer is simple and effective. That said Twinkle Tray is clearly the more advanced and polished software. I just thought about adding a hotkey for adjusting my brightness. Already implemented in Twinkle Tray!
I thought about an easy option, which wouldn't clutter anything for users which don't need this feature. You can add a toggle in the settings named e.g. "Add slider to go below 0 brightness".
Now all slider don't go from 0 to 100, instead they reach from -100 to 100.
If adjusting brightness below 0, the software switches from the DDC / CI method to a filter method. Probably the same Dimmer uses. Every monitor would support it as it's just a filter layed on top.
